### ISSUE TITLE
Blog: Mapping Locations on Dashboard 2.0

### ISSUE THREAD

**Issue by sumitshinde-84** [FlowFuse/website#2015]
### Description

Write a blog post detailing how to obtain location data (latitude, longitude) and plot it on a map within Dashboard 2.0.

**Comment by joepavitt** [comment]
What needs the additional work for multiple items?

Each code can be setup to render at a different path, and therefore, the iframe can be adjusted accordingly

**Comment by ZJvandeWeg** [comment]
@joepavitt I think it's still valuable to blog about how to track say busses in a city on that worldmap. So multiple items (busses) need to be plotted on the map, and updated later. Or any other item that works well for a blog.

**Comment by joepavitt** [comment]
Yes, sorry, thought you meant functionally, we were missing something on the Dashboard side.

All of that would be a general purpose "how to use worldmap" tutorial, but displayed in  a Dashboard 2.0 UI

**Comment by sumitshinde-84** [comment]
Why can't we plot multiple items on the map?

**Comment by joepavitt** [comment]
We can @sumitshinde-84 - just use the worldmap node

**Comment by sumitshinde-84** [comment]
> We can @sumitshinde-84 - just use the worldmap node

![Screenshot (610)](https://github.com/FlowFuse/website/assets/110285294/c39f09c8-d69d-4fd7-abcd-b06b07875a70)
